### PR TITLE
Show user email in error messages

### PR DESCRIPTION
**Pull request by skyfallwastaken**
This PR adds the user's email to the Nest error message instead of their @hackclub.app one.

### CODE REVIEW

**Comment by dispherical** [comment]
They do not.

Plus there is a privacy risk of putting user emails out there.

**Review by NeonGamerBot-QK — CHANGES_REQUESTED** [review]
Where would the user instance be created in the Db? the code shows no sign of creating an instance for a user.

> note: i am not a maintainer of the nest so you can dismiss this review;

**Review by NeonGamerBot-QK — APPROVED** [review]
lgtm
> note: i am not a maintainer of the nest so my review does not matter that much;

File: cabinet/utils.ts [hunk 1]
```diff
@@ -123,6 +124,20 @@ export async function reloadCaddy() {
...
     });
 
+    let specifiedEmail = null;
+    try {
+      specifiedEmail = fs.readFileSync(
+        `/home/${domain.username}/.error-page-email`,
+        "utf8",
+      );
+    } catch (err) {
+      console.error("Failed to read specified email:", err);
```
Comment (hello-smile6) [inline cabinet/utils.ts]: This would spam logs a lot for no reason

Comment (skyfallwastaken) [reply cabinet/utils.ts]: We now only throw errors if the error isn't a "file not found" error.

**Review by polypixeldev — APPROVED** [review]
when merged, pls update the guides to document this!

**Comment by skyfallwastaken** [comment]
@polypixeldev should I just add it to the caddy section or should I make a new page?

**Review by dispherical — APPROVED** [review]

**Comment by dispherical** [comment]
> @polypixeldev should I just add it to the caddy section or should I make a new page?

I'd suggest the Caddy page since it doesn't need a whole page to explain it

**Pull request merged**